### DISCUSSION TITLE
Is toggling widgets button on taskbar disabled?

### DISCUSSION THREAD

**Discussion by n7alam** [valinet/ExplorerPatcher#113] [Q&A] [answered]
I can use Win+W to open widgets pane, but there used to be a button for this on the taskbar. Now it is gone and, unless I'm missing something, I can't get the button to appear through toggling it on Windows settings. I don't really mind the button's disappearance but was wondering if this is intentional or not?

**Comment by valinet** [comment, marked as answer]
Hi

Yeah, it kind of is. Not directly as a “feature”, but you see, Windows 10 never had such a button, so nothing to “revive” from the old code and display for that… idk, maybe we could make a small app that we’d pin and when launched it presses Win+W and triggers the widgets panel… an example, just a thought.

Thanks

**Reply by n7alam** [reply]
I do believe one update for Windows 10 included a [lite version](https://www.howtogeek.com/734719/how-to-get-windows-10s-taskbar-weather-widget-back/) of this where the weather was reported in the taskbar and you could click on it to display a flyout similar to the widget in Windows 11. I don't particularly miss this button, but it makes sense now why it is not there. Thanks!

**Reply by valinet** [reply]
Yeah, indeed, that News and features thing. They also removed that altogether, there’s nothing to enable in the code, as far as I know. Or at least, I don’t know of any of that to still be in there, at the moment.
